README.md

```diff
@@ -1,24 +1,95 @@
+
+![The Lower Thirds card with the color modal open](images/gjallarcolors.png)
+
+If a card is unnecessary or temporarily unneeded, you can minimize it by clicking the arrow at the right of each card next to the push button.
+
+These JSON files can be connected to broadcasting software (vMix or OBS with applicable plugins) to allow for seamless editing and usage of data. However, not all of this data will be useful or even applicable even if you are hosting a tournament.
+
+### Header
+
+In the top left corner, the current tournament is indicated next to the hamburger menu. The pencil allows you to change the active tournament. Enter the tournament’s unique slug (“brawlhalla-world-championship-expo-2023”) to switch to that tournament. Shorthand forms also exist for certain tournaments (“bcx”). Clicking on the game mode (In this example, Brawlhalla Championship 1v1) allows you to swap between different tournament game modes if they exist. The pools can also be swapped similarly.
+
+Push Brackets will output JSON to be used in bracket images. You can have these pushed every two minutes by clicking the timer next to the button.
+
+![A green checkmark](images/gjallarcheck.png)
+
+Hovering over the checkmark icon next to the search bar shows you the application latency and the rate limits for both the database and start.gg.
+
+![A yellow disconnection indicator](images/gjallardc.png)
+
+If your Gjallarhorn instance is unexpectedly terminated, the icon will change and turn yellow.
+
+![An active user icon](images/gjallaruser.png)
+
+Hovering over this icon will show checkboxes that enable light mode and push notifications for possible errors respectively.
+
+### Casters
+
+![The Casters card](images/gjallarcasters.png)
+
+You can enter up to four different casters, with text boxes for their name, Twitter/X accounts, and pronouns.
+
+### Game
+
+![The Game card](images/gjallargame.png)
+
+This card can be input manually, or you can use the Set dropdown to populate text with a start.gg match. You can also import a set from the Queue card by clicking on the clipboard next to Push Round. Rounds won can be incremented using the plus and minus buttons next to Left and Right.
+
+### Players
+
+![The Player card](images/gjallarplayers.png)
+
+This card can be input manually. You can also import players from the Game card or the Queue card with the clipboard button next to Push. When importing players, Gjallarhorn will attempt to populate the Lifetime Score textbox with information from the Stats API. It can also be manually incremented with the plus and minus buttons. The legend loaded for each player will be the most recently reported legend in the current tournament (if applicable). Otherwise, it will be the last legend the player ever reported.
+
+### Ticker
+
+![The Ticker Card](images/gjallarticker.png)
+
+Can be used for miscellaneous ticker text. By clicking the timer next to shuffle, the tickers will automatically shuffle every five minutes.
```

Comment:
No reason to call the ticker text miscellaneous, we could have instances where tournament producers would consider the ticker info extremely non-miscellaneous.

Reply:
Miscellaneous is the perfect word wym

![image](https://github.com/BMG-Esports/gjallarhorn/assets/13533898/be404abf-be14-4d4b-9979-5f0b9a5d34fb)

Reply:
Went ahead and changed it anyway, while also adding a bit more information to the card. Not sure if miscellaneous is important enough to split hairs over.